package.json

```diff
@@ -30,6 +30,7 @@
     "vue-tsc": "1.0.24"
   },
   "dependencies": {
+    "eslint-plugin-tailwindcss": "^3.10.1",
```

Comment:
`"eslint-plugin-tailwindcss": "3.10.1"` 🙏 
We've been following the convention of using exact versions of packages installed

Reply:
sure thing 👍🏻